OptimoveSDK/Sources/Classes/Optimove.swift

```diff
@@ -130,6 +130,66 @@ public extension Optimove {
     @objc static func reportEvent(_ event: OptimoveEvent) {
         shared.reportEvent(event)
     }
+
+    @available(iOS 13.0, *)
+    func getPreferences(brandGroupId: String) async throws -> Preferences {
+        return try await withCheckedThrowingContinuation { continuation in
+            container.resolve { serviceLocator in
+                Task {
+                    do {
+                        let preferences = try await serviceLocator.preferenceCenter().getPreferences(
+                            brandGroupId: brandGroupId
+                        )
+                        continuation.resume(returning: preferences)
+                    } catch {
+                        continuation.resume(throwing: error)
+                    }
+                }
+            }
+        }
+    }
+
+
+    @available(iOS 13.0, *)
+    @objc static func getPreferences(brandGroupId: String) async throws -> Preferences {
```

Comment:
I think objc does not support modern async. Need to be checked. 
This way, we might avoid using objc for async funcs